### PR TITLE
fix(table-footer): added styling according to designer feedback

### PR DESCRIPTION
**Pull request by Lunkan89**
## **Describe pull-request**  
Made changes according to designer feedback

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3127](https://tegel.atlassian.net/browse/CDEP-3127)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to...
2. Check in...
3. Run ...

## **Checklist before submission**
- [ ] I have added unit tests for my changes (if applicable)
- [ ] All existing tests pass
- [ ] I have updated the documentation (if applicable)
- [ ] Not breaking production behavior
- [ ] Behavior available in storybook with documented descriptions (if applicable)
- [ ] `npm run build-all` without errors

## **Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

## **Screenshots**  
Darkmode:
Closed:
Screenshot 2024-10-25 at 15 11 06

Opened:
Screenshot 2024-10-25 at 15 11 10

Lightmode:
Closed:
Screenshot 2024-10-25 at 15 10 51

Opened:
Screenshot 2024-10-25 at 15 10 58

[CDEP-3127]: https://tegel.atlassian.net/browse/CDEP-3127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ckrook — APPROVED** [review]
Looks good Johan! ⭐

File: packages/core/src/components/dropdown/dropdown.scss [hunk 1]
```diff
@@ -23,13 +23,32 @@
   .dropdown-select {
     position: relative;
 
+    button:focus {
+      border-bottom-color: var(--tds-dropdown-border-bottom);
```
Comment (nathalielindqvist) [inline packages/core/src/components/dropdown/dropdown.scss:27]: I haven't been part of the discussion around these updated styles, should the dropdown not have any blue border when focused like the input field for number of pages?

File: packages/core/src/components/dropdown/dropdown-vars.scss [hunk 2]
```diff
@@ -46,7 +52,11 @@
   }
 
   tds-table-footer {
-    --tds-dropdown-bg: transparent;
+    --tds-dropdown-bg: var(--tds-grey-868);
     --tds-placeholder-margin: var(--tds-spacing-element-8);
+    --tds-dropdown-border-bottom: transparent;
+    --tds-dropdown-border-bottom-hover: transparent;
+    --tds-dropdown-border-bottom-open: transparent;
+    --tds-dropdown-border-radius: var(--tds-spacing-element-4);
```
Comment (mJarsater) [inline packages/core/src/components/dropdown/dropdown-vars.scss:60]:
```suggestion
    --tds-dropdown-border-radius: var(--tds-dropdown-border-radius);
```

File: packages/core/src/components/dropdown/dropdown.scss (hunk 1 above)
Comment (theJohnnyMe) [reply packages/core/src/components/dropdown/dropdown.scss:27]: Good call. Designs for this are in "WIP" so we do guesses here, but I will take it on myself and add a blue outline like the input on the right side has.

**Review by nathalielindqvist — APPROVED** [review]
Looking good! 🌟

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Pull request merged**